test/unit/usergroup_test.rb

```diff
@@ -100,8 +92,8 @@ def populate_usergroups
   end
 
   test "removes user join model records" do
-    ug1 = Usergroup.find_or_create_by_name :name => "ug1"
-    u1  = User.find_or_create_by_login :login => "u1", :mail => "u1@someware.com", :auth_source => auth_sources(:one)
+    ug1 = Usergroup.where(:name => "ug1").first_or_create
+    u1  = FactoryGirl.build(:user)
```

Comment:
I think this is meant to simply be `@u1`, it would have found the u1 record in the old code.  I think it's testing that removing `u2` removes the UsergroupMember.